pom.xml

```diff
@@ -936,6 +937,12 @@
                 <artifactId>jsoup</artifactId>
                 <version>${jsoup.version}</version>
             </dependency>
+            <!-- iotdb -->
+            <dependency>
+                <groupId>org.apache.iotdb</groupId>
```

Comment:
dependency already exists

https://github.com/apache/incubator-seatunnel/blob/dev/pom.xml#L947

Reply:
I guess it's because I used the github page to resolve the conflict, but ignored this, I'll rework it